server/api/controllers/webhook/getData.ts

```diff
@@ -0,0 +1,25 @@
+import { firestore } from './firebase';
+import { collection, query, getDocs } from '@firebase/firestore';
+const ref = collection(firestore, 'clients');
+type Record = { board_ID: string; username: string; api_key: string };
+const allData: Record[] = [];
+
+const getData = async () => {
```

Comment:
this an unhelpful name for this function.

Reply:
changed the function name to `getUsersRecord`